Makefile

```diff
@@ -24,20 +24,18 @@ PERSONDB=FORCE
 ##  The numbers given here RULE
 ##
 
-##  Current series
-SERIES=1.1.1
+##  Current series.  Variable names are numbered to indicate:
+##  SERIES1	OpenSSL pre-3.0
+##  SERIES3	OpenSSL 3.0 and on
+SERIES1=1.1.1
+SERIES3=3.0
 ##  Older series.  The second type is for source listings
 OLDSERIES=1.1.0 1.0.2 1.0.1 1.0.0 0.9.8 0.9.7 0.9.6
 OLDSERIES2=1.1.0 1.0.2 1.0.1 1.0.0 0.9.x
 ##  Series for manual layouts
 MANSERIES1=1.1.1
 MANSERIES3=3.0
 
-##  Future series, i.e. a series that hasn't had any final release yet.
-##  We distinguish them to avoid having to produce notes, vulnerability
-##  documents, ...
-FUTURESERIES=3.0
```

Comment:
Might we want this in the future?  I.e. does deleting it now cause more work then?

Reply:
Yeahok, I can put it back, empty.